### PR TITLE
Added missing parameters to GitHub app manifest.

### LINKED ISSUE #22315: Parameters missing from GitHub App Manifests
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/enterprise-server@3.6/developers/apps/building-github-apps/creating-a-github-app-from-a-manifest#github-app-manifest-parameters

### What changes are you suggesting?

The "GitHub App Manifest parameters" section should be updated to include all possible parameters.

I haven't tried myself, but it seems likely that `request_oauth_on_install` and `setup_on_update` are supported and undocumented manifest parameters.

Request to open this issue: https://github.com/github/docs/issues/21659#issuecomment-1307841708

### Additional information

Here's a comparison between the parameters listed in the manifest documentation versus the URL parameters documentation.

|Parameter|[URL parameters](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app-using-url-parameters#github-app-configuration-parameters)|Manifest parameters|
|--|:-:|:-:|
|`name`|✅|✅|
|`description`|✅|✅|
|`url`|✅|✅|
|`callback_urls`|✅|✅|
|`public`|✅|✅|
|`request_oauth_on_install`|✅|❌|
|`setup_url`|✅|✅|
|`setup_on_update`|✅|❌|
|`webhook_active`|✅|❌ see: `hook_attributes`|
|`webhook_url`|✅|❌ see: `hook_attributes`|
|`hook_attributes`|❌|✅|
|`events`|✅|❌ see: `default_events`|
|`default_events`|❌ see: `events`|✅|
|`single_file_name`|✅|❌|
|`single_file_paths`|✅|❌|
|`redirect_url`|❌|✅|
|`default_permissions`|❌ passed in as [key-value pairs](https://docs.github.com/en/developers/apps/building-github-apps/creating-a-github-app-using-url-parameters#github-app-permissions)|✅|

### PR DESCRIPTION
**Pull request by OlayinkaAtobiloye**
### Why:

Closes [ISSUE](https://github.com/github/docs/issues/22315)

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Demo of changed issue.

### Check off the following:

- [x] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
👋 Hey @OlayinkaAtobiloye! 🎉 🥳 

I'll get this triaged for writer review! 💖

File: content/developers/apps/building-github-apps/creating-a-github-app-from-a-manifest.md [hunk 1]
```diff
@@ -67,6 +67,8 @@ The person creating the app will be redirected to a GitHub page with an input fi
 `public` | `boolean` | Set to `true` when your GitHub App is available to the public or `false` when it is only accessible to the owner of the app.
 `default_events` | `array` | The list of [events](/webhooks/event-payloads) the GitHub App subscribes to.
 `default_permissions` | `object` | The set of [permissions](/rest/reference/permissions-required-for-github-apps) needed by the GitHub App. The format of the object uses the permission name for the key (for example, `issues`) and the access type for the value (for example, `write`).
+`request_oauth_on_install` | `boolean` | Set to `true` to request the user to authorize the GitHub App, after the GitHub App is installed.
```
Comment (sophietheking) [inline content/developers/apps/building-github-apps/creating-a-github-app-from-a-manifest.md:70]:
```suggestion
`request_oauth_on_install` | `boolean` | Set to `true` to request the user to authorize the {% data variables.product.prodname_github_app %}, after the {% data variables.product.prodname_github_app %} is installed.
```

You can replace "GitHub App" with a variable (the string with the curly brackets) here. ✨  This makes product name changes easier to implement across the site and avoids typos in our product names. 🙇‍♀️  You can find the product variables in `/data/variables/product.yml`.

File: content/developers/apps/building-github-apps/creating-a-github-app-from-a-manifest.md [hunk 2]
```diff
@@ -67,6 +67,8 @@ The person creating the app will be redirected to a GitHub page with an input fi
...
+`setup_on_update` | `boolean` | Set to `true` to redirect users to the `setup_url` after they update your GitHub App installation.
```
Comment (sophietheking) [inline content/developers/apps/building-github-apps/creating-a-github-app-from-a-manifest.md:71]:
```suggestion
`setup_on_update` | `boolean` | Set to `true` to redirect users to the `setup_url` after they update your {% data variables.product.prodname_github_app %} installation.
```

Same as above! 💖

**Review by sophietheking — DISMISSED** [review]
Congrats on your first Docs PR! ✨🕺⚡🎉  I left two small suggestions, but this looks really good to me! 

Feel free to `@mention` me here once you've looked at my suggestions and I'll get this merged for you.

File: content/developers/apps/building-github-apps/creating-a-github-app-from-a-manifest.md (hunk 2 above)
Comment (OlayinkaAtobiloye) [reply content/developers/apps/building-github-apps/creating-a-github-app-from-a-manifest.md]: Hi, @sophietheking, thanks for the review!, super helpful I have made the changes now.

**Review by sophietheking — APPROVED** [review]
Looking good! Thanks for updating the other instances of "GitHub App" as well. 💖

**Review by sophietheking — APPROVED** [review]
Looking good! Thanks for updating the other instances of "GitHub App" as well. 💖

**Pull request merged**